README.md

```diff
@@ -1,2 +1,40 @@
-# wrixby-ecom
- 
+<div align="center">
+  
+ # Wrixby ECom
+  
+ ### An Online Store for Bags and Wallet Shopping.
+  
+ [![Netlify Status](https://api.netlify.com/api/v1/badges/36ad02b6-d66b-4b21-b037-33bd2b9e34d6/deploy-status)](https://app.netlify.com/sites/wrixby-ecom/deploys)
+
+<a href="https://wrixby-ecom.netlify.app/"><strong>Explore Wrixby ECom</strong></a>
+
+</div>
+
+## What's included:
+
+1. [Home Page](https://wrixby-ecom.netlify.app/)
+1. [Product Page](https://wrixby-ecom.netlify.app/product/)
+1. [Cart](https://wrixby-ecom.netlify.app/cart/)
+1. [Wishlist](https://wrixby-ecom.netlify.app/wishlist/)
+1. [Login](https://wrixby-ecom.netlify.app/auth/login.html)
+1. [Sign Up](https://wrixby-ecom.netlify.app/auth/signup.html)
+
+## Demo
+
+![App Screenshot](https://wrixbyui.netlify.app/assets/images/Wrixby ECom.png)
+
```

Comment:
Your screenshot link is not working. Please look into it. Everything else looks good.

Reply:
It's added in this same PR, will work after merge